otherPages/type/typeFood.html

```diff
@@ -44,6 +44,35 @@
         
     </section>
     <!-- End Cards Section -->
+    <!-- Recipe popup -->
+    <div class="overlay">
+        <div class="pop-up"> 
```

Comment:
I'm good with vertical scroll but horizontal one is messy, ask your team for suggestions